app/filtering.js

```diff
@@ -511,7 +511,7 @@ function registerPermissionHandler (session, partition) {
             {text: locale.translation('deny')},
             {text: locale.translation('allow')}
           ],
-          frameOrigin: getOrigin(mainFrameUrl),
+          frameOrigin: getOrigin(origin),
```

Comment:
i think you want mainFrameUrl here

Reply:
The variable names  need to be updated here. 

In https://github.com/brave/muon/pull/540. I updated the `origin` to be the security origin of the current frame, and the `mainFrameUrl` to the URL requesting the permission.